diff --git a/index.d.ts b/index.d.ts
index 4bf09a1cb272e4ff28334dd524057afdc36980e7..580b2ac4021d8d4ba75883578304364f37e2d044 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,2 +1,2 @@
-export { ReadOnlyPropertiesMixin } from './src/read-only-properties-mixin';
-export { spread, spreadEvents, spreadProps } from './src/spread';
\ No newline at end of file
+export { ReadOnlyPropertiesMixin } from './src/read-only-properties-mixin.js';
+export { spread, spreadEvents, spreadProps } from './src/spread.js';
\ No newline at end of file
diff --git a/src/spread.d.ts b/src/spread.d.ts
index e93e5cc5dcfe4af6d505cd38ae2af12f4ecf49ae..e44fdbe19b2cc0e7a601a56bea7be0918835825b 100644
--- a/src/spread.d.ts
+++ b/src/spread.d.ts
@@ -40,7 +40,7 @@ export declare class SpreadPropsDirective extends AsyncDirective {
 }
 export declare const spreadProps: (_spreadData: {
     [key: string]: unknown;
-}) => import("lit-html/directive").DirectiveResult<typeof SpreadPropsDirective>;
+}) => import("lit-html/directive.js").DirectiveResult<typeof SpreadPropsDirective>;
 /**
  * Usage:
  *    import { html, render } from 'lit';
@@ -78,7 +78,7 @@ export declare class SpreadEventsDirective extends SpreadPropsDirective {
 }
 export declare const spreadEvents: (_spreadData: {
     [key: string]: unknown;
-}) => import("lit-html/directive").DirectiveResult<typeof SpreadEventsDirective>;
+}) => import("lit-html/directive.js").DirectiveResult<typeof SpreadEventsDirective>;
 /**
  * Usage:
  *    import { html, render } from 'lit';
@@ -108,6 +108,6 @@ export declare class SpreadDirective extends SpreadEventsDirective {
 }
 export declare const spread: (_spreadData: {
     [key: string]: unknown;
-}) => import("lit-html/directive").DirectiveResult<typeof SpreadDirective>;
+}) => import("lit-html/directive.js").DirectiveResult<typeof SpreadDirective>;
 export {};
 //# sourceMappingURL=spread.d.ts.map
\ No newline at end of file
